Library/ViewModels/LiveStreamContainerViewModel.swift

```diff
@@ -166,8 +207,10 @@ LiveStreamContainerViewModelInputs, LiveStreamContainerViewModelOutputs {
         .take(first: 1)
     )
 
-    self.projectImageUrl = project
-      .map { URL(string: $0.photo.full) }
+    self.projectImageUrl = project.flatMap { project in
+      SignalProducer(value: URL(string: project.photo.full))
```

Comment:
@mbrandonw ok so this fixes it and the tests pass. I know using a `SignalProducer` might not be the desired approach but it's handy to be able to `prefix` it with `nil` and makes what we're wanting to do pretty clear?

I could not get `nil` to emit first using `viewDidLoad` without a nested `combineLatest` which I also didn't like.

And I think having this testable is safer than only removing the image from the story board in case it was ever added back?